libbeat/scripts/generate_fields_docs.py

```diff
@@ -90,26 +89,28 @@ def fields_to_asciidoc(input, output, beat):
 
     sections = docs["sections"]
 
+    # Add default section
+    sections["beat"] = "Beat"
+
     # Check if sections is define
     if sections is None:
         print "No sections are defined in fields.yml. fields.asciidoc cannot be generated."
         return
 
-    for doc, _ in sections:
-        output.write("* <<exported-fields-{}>>\n".format(doc))
-    output.write("\n")
 
-    for value in sections:
+    for key in sorted(sections):
+        output.write("* <<exported-fields-{}>>\n".format(key))
+    output.write("\n")
 
-        doc = value[0]
-        name = value[1]
+    for key in sorted(sections):
```

Comment:
A sorted it as this seems to me more a reference so alphabetical look should be easiest I was thinking. 

About the second part: So you mean instead of using sections, all the information is already read form the existing entries. In any case I like the idea.